src/help.h

```diff
@@ -1,4 +1,4 @@
-/* Automatically generated by utils/generate-command-help.rb, do not edit. */
+/* Automatically generated by ./utils/generate-command-help.rb, do not edit. */
```

Comment:
did you re-generate that file producing no changes?
i guess because the script you used samples the old copy of commands.json?
maybe make the extra effort to give it a temp path so we can update both in the same commit?

Reply:
See 56613896e